app/iot/devices.py

```diff
@@ -1,64 +1,78 @@
-import time
+import asyncio
 
-from .message import MessageType
+from app.iot.message import MessageType
```

Comment:
Ensure that the `MessageType` class or enum is correctly defined in the `app.iot.message` module. This import is crucial for the `send_message` method in each device class.